### PR TITLE
i have added a good place to get wood

### PR DESCRIPTION
**Pull request by nawfaljahib**
see location in changes!!

### CODE REVIEW

**Review by TheFar4 — COMMENTED** [review]
yeah it looks good, i will include it - just kidding its a test

make the place anme more descriptive, is it plaisance airport?

**Comment by nawfaljahib** [comment]
ok

File: index.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ You can use the [editor on GitHub](https://github.com/TheFar4/MakerResourcesMaur
 
 Whenever you commit to this repository, GitHub Pages will run [Jekyll](https://jekyllrb.com/) to rebuild the pages in your site, from the content in your Markdown files.
 
+A good place to get wood is plaisance
```
Comment (nawfaljahib) [inline index.md:7]: the place name will be replaced

**Comment by TheFar4** [comment]
please make changes and remake your request

**Pull request closed without merging**